src/ResourceManager/LogicApp/Commands.LogicApp/Cmdlets/IntegrationAccount/GetAzureIntegrationAccountAgreementCommand.cs

```diff
@@ -32,9 +32,8 @@ public class GetAzureIntegrationAccountAgreementCommand : LogicAppBaseCmdlet
         public string ResourceGroupName { get; set; }
 
         [Parameter(Mandatory = false, HelpMessage = "The integration account name.")]
-        [Alias("ResourceName")]
         [ValidateNotNullOrEmpty]
-        public string Name { get; set; }
+        public string IntegrationAccountName { get; set; }
```

Comment:
@pankajsn this is breaking change. You cannot do this. You will first have to add a warning. And when we have a breaking change release at that time you will be able to make this change. This cannot happen in this release.